pkgs/development/compilers/dtc/default.nix

```
@@ -12,14 +12,14 @@ stdenv.mkDerivation rec {
     sha256 = "1jhhfrg22h53lvm2lqhd66pyk20pil08ry03wcwyx1c3ln27k73z";
   };
 
-  nativeBuildInputs = [ flex bison pkgconfig which ] ++ lib.optionals pythonSupport [ python2 swig ];
-  buildInputs = lib.optionals pythonSupport [ python2 ];
+  nativeBuildInputs = [ flex bison pkgconfig which ] ++ lib.optionals pythonSupport [ python swig ];
+  buildInputs = lib.optionals pythonSupport [ python ];
 
   postPatch = ''
     patchShebangs pylibfdt/
   '';
 
-  makeFlags = lib.optionals (!pythonSupport) [ "NO_PYTHON=1" ];
+  makeFlags = [ "PYTHON=python" ];
   installFlags = [ "INSTALL=install" "PREFIX=$(out)" "SETUP_PREFIX=$(out)" ];
```

I know you didnt change it, but now would be the time
```suggestion
  installFlags = [ "INSTALL=install" "PREFIX=${placeholder "out"}" "SETUP_PREFIX=${placeholder "out"}" ];
```